cmd/tempo/main.go

```diff
@@ -92,17 +92,7 @@ func main() {
 	// Allocate a block of memory to alter GC behaviour. See https://github.com/golang/go/issues/23044
 	ballast := make([]byte, *ballastMBs*1024*1024)
 
-	// Warn the user for suspect configurations
-	if warnings := config.CheckConfig(); len(warnings) != 0 {
```

Comment:
should we just move this logic into `loadConfig()` instead of calling it in main and loadConfig()? i'm not sure why it's here.